usvm-core/src/main/kotlin/org/usvm/model/Model.kt

```diff
@@ -37,9 +38,10 @@ open class UModelBase<Type>(
     override val mocker: UMockEvaluator,
     val regions: Map<UMemoryRegionId<*, *>, UReadOnlyMemoryRegion<*, *>>,
     val nullRef: UConcreteHeapRef,
+    override val ownership: MutabilityOwnership = ctx.defaultOwnership,
```

Comment:
It is better to use fresh (new) ownership here instead of default. Default ownership is ok only if we can guarantee that the mutable write can't corrupt anything.